File: 2116-count-number-of-pairs-with-absolute-difference-k/count-number-of-pairs-with-absolute-difference-k.java
class Solution {
    public int countKDifference(int[] nums, int k) {
        int count=0;
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                int x=nums[i];
                int y=nums[j];
                int w=Math.abs(x-y);
                if(w==k){
                    count+=1;
                }
            }
        }
        return count;

    }
}

File: 2116-count-number-of-pairs-with-absolute-difference-k/count-number-of-pairs-with-absolute-difference-k.py
class Solution:
    def countKDifference(self, nums: List[int], k: int) -> int:
        count=0
        for i in range(0,len(nums)):
            for j in range(i+1,len(nums)):
                x,y=nums[i],nums[j]
                w=abs(x-y)
                if w==k:
                    count+=1
        return count